frontend/mixins/signedExtrinsic.js

```diff
@@ -0,0 +1,166 @@
+import { gql } from 'graphql-tag'
+export default {
+  methods: {
+    async extrinsicsDayCount(queryLimit) {
```

Comment:
Rename all methods so it is clear that it is about signed trxs

Reply:
Done